app/services/notices/setup/fetch-notifications.service.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+/**
+ * Fetch notifications for a notice
+ * @module FetchNotificationsService
+ */
+
+const NotificationModel = require('../../../../app/models/notification.model.js')
+
+/**
+ * Fetch notifications for a notice
+ *
+ * The legacy code used a table called 'events' which groups all the notifications together. This is why we pass in the
+ * eventId as this relates to the 'event_id' column.
+ *
+ * @param {string} eventId - the unique id used to group all the notifications together
+ *
+ * @returns {object} - the notifications for the event
+ */
+async function go(eventId) {
+  return NotificationModel.query().where('event_id', eventId)
```

Comment:
If using Objection.js you should be able to just use camelcase for properties.


```suggestion
  return NotificationModel.query().where('eventId', eventId)
```

However, my main request is that we specify the fields we want, even if it is all of them (though I'd be surprised if it was), at this time.

Therefore, we need to add `.select()` to the query.

Reply:
fair point well made. 

I will look into what we need and update.